docs/table/index.rst

```diff
@@ -44,98 +39,93 @@ case, while the full `astropy.table` documentation is available from the
 First create a simple table with three columns of data named ``a``, ``b``,
 and ``c``.  These columns have integer, float, and string values respectively::
 
-  >>> from astropy.table import Table
+  >>> from astropy.table import QTable
+  >>> import astropy.units as u
   >>> a = [1, 4, 5]
   >>> b = [2.0, 5.0, 8.2]
   >>> c = ['x', 'y', 'z']
-  >>> t = Table([a, b, c], names=('a', 'b', 'c'), meta={'name': 'first table'})
+  >>> t = QTable([a, b, c], names=('a', 'b', 'c'), meta={'name': 'first table'})
```

Comment:
The point is that, apart from one extra character in the name, `QTable` is not overkill and there is nothing more expensive about it.  But I see your point and that's why I thought it might be controversial.  Maybe we need to wait for @mhvk (I recall that he was going on vacation though).  If beginners just start using `QTable` by default, they are more likely to use `Quantity` and units!

Reply:
(on my way back from vacation, waiting for a delayed flight...) I like the idea of going straight to `QTable` in the getting-started section, but also agree with @pllim that it is perhaps a bit confusing without an actual quantity. So, perhaps just make `b` a `Quantity`?